backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Window.java

```diff
@@ -163,10 +163,11 @@ public void run() {
 		this.tmpBuffer2 = BufferUtils.createIntBuffer(1);
 	}
 
-	void create(long windowHandle) {
```

Comment:
As long as we know the old `Lwjgl3Window.create(long)` was only used here (which is likely but not certain, since it's package-private and could be accessed by reflection), this probably won't break anything. It isn't a public API, so we don't need to maintain any sort of compatibility, but as long as this builds it means changing the method signature won't be an issue, I think.